proseco/tests/test_catalog.py

```diff
@@ -248,6 +248,25 @@ def test_bad_obsid():
     assert aca.fids.thumbs_up == 0
 
 
+def test_check_guide_thumbs_for_ers():
+    dark = DARK40.copy()
+    stars1 = StarsTable.empty()
+    stars1.add_fake_constellation(mag=[8.0, 8.0, 8.0, 10.0, 10.0, 10.0, 10.0, 10.0],
```

Comment:
Should update this to use the thresholds in characteristics if we decide to go with something like this change.